tests/lib/outputs/html/html_test.py

```diff
@@ -15,7 +15,9 @@
 
 html_stats = {
     "total_pass": 0,
+    "total_muted_pass": 0,
     "total_fail": 1,
+    "total_muted_fail": 0,
```

Comment:
The current HTML had a bug, please add a value different than the default one to catch it.
```suggestion
    "total_muted_fail": 10,
```

Reply:
done